rwt_image_view/www/index.html

```diff
@@ -30,14 +30,14 @@ <h1><span class="glyphicon glyphicon-star"></span>rwt_image_view <small>robotweb
       </div>
     </div>
 
-    <script type="text/javascript" src="../rwt_utils_3rdparty/jquery/jquery.min.js"></script>
-    <script type="text/javascript" src="3rdparty/ros/js/eventemitter2.js"></script>
-    <script type="text/javascript" src="3rdparty/ros/js/roslib.js"></script>
-    <script type="text/javascript" src="../rwt_utils_3rdparty/lodash.min.js"></script>
-    <script type="text/javascript" src="../rwt_utils_3rdparty/bootstrap/js/bootstrap.min.js"></script>
-    <script type="text/javascript" src="../rwt_utils_3rdparty/mjpegcanvasjs/build/mjpegcanvas.min.js"></script>
-    <script type="text/javascript" src="../rwt_utils_3rdparty/rwt_utils.js"></script>
-    <script type="text/javascript" src="js/rwt_image_view_main.js"></script>
+    <script type="text/javascript" src="/rwt_utils_3rdparty/jquery/jquery.min.js"></script>
+    <script type="text/javascript" src="/rwt_image_view/3rdparty/ros/js/eventemitter2.js"></script>
+    <script type="text/javascript" src="/rwt_image_view/3rdparty/ros/js/roslib.js"></script>
```

Comment:
https://github.com/tork-a/visualization_rwt/pull/77/files changed absolute path to relative path. Which is correct?
https://github.com/tork-a/visualization_rwt/blob/959bf918d6c9d110fbf01a11298e0facafe1bf30/rwt_robot_monitor/www/plotter.html#L41
https://github.com/tork-a/visualization_rwt/pull/77/files#r259242919

Reply:
I'm writing the issue details here. I'm not familiar with javascript, so I don't know which one is better.
But in order to solve the issue, I use absolute path, but if we can solve the issue with relative path, we can use relative path probably.

https://github.com/tork-a/visualization_rwt/pull/91#issuecomment-649978444